device-types/Lenovo/SR650.yaml

```diff
@@ -14,3 +16,6 @@ power-ports:
     type: iec-60320-c14
   - name: PSU1
     type: iec-60320-c14
+console-ports:
+  - name: Rear VGA
```

Comment:
VGA ports are not meant to be modelled this way, lets remove the console-ports/vga adapters

Reply:
removed